### PR TITLE
RDoc-699 more common errors

### PR DESCRIPTION
**Pull request by iftahbe**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown [hunk 1]
```diff
@@ -81,6 +81,53 @@ If this happens, there is nothing you can do except wait for DNS propogation. Wh
...
+
+You're trying to import the client certificate received from RavenDB to Firefox but get the following error:
+
+![Figure 3. Unknown Reasons](../../server/security/images/3.png)
+
+{CODE-BLOCK:plain}
+The PKCS#12 operation failed for unknown reasons.
+{CODE-BLOCK/}
+
+Firefox **sometimes** fails to import a certificate which is not password protected.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown]: Did we report that to Firefox team?

File: Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown [hunk 2]
```diff
@@ -81,6 +81,53 @@ If this happens, there is nothing you can do except wait for DNS propogation. Wh
...
+To overcome this issue, use the RavenDB Cli to [generate a password protected certificate](../../server/administration/cli#generateclientcert) or add a password to the current certificate by importing it to the OS store and exporting it back with a password.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown]: Cli -> CLI

File: Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown [hunk 3]
```diff
@@ -81,6 +81,53 @@ If this happens, there is nothing you can do except wait for DNS propogation. Wh
...
+
+Firefox **sometimes** fails to import a perfectly good certificate for no good reason and fails to give you a proper error. You can try to generate a new password protected certificate using the RavenDB Cli and import that instead.
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown]: Cli -> CLI

File: Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown (hunk 1 above)
Comment (iftahbe) [reply Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown]: Many people report this in many threads for years... example https://bugzilla.mozilla.org/show_bug.cgi?id=1049435

The interesting part is that the same certificates which fail on Firefox work in all other tools including openssl, chrome, edge

Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/server/security/common-errors-and-troubleshooting.markdown]: "Reported: 4 years ago" :)

**Pull request merged**